deepmd/pt/model/descriptor/sezm.py

```
@@ -1510,13 +1684,27 @@ def _init_lm_schedules(
             raise ValueError("`m_schedule` must have the same length as `l_schedule`")
         if any(x < 0 for x in self.m_schedule):
             raise ValueError("`m_schedule` entries must be non-negative")
-        if any(m > l for m, l in zip(self.m_schedule, self.l_schedule)):
+        if any(m > l for m, l in zip(self.m_schedule, self.l_schedule, strict=True)):
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)